pilot/test/util/kubernetes.go

```diff
@@ -105,6 +105,14 @@ func describeNotReadyPods(items []v1.Pod, kubeconfig, ns string) {
 	}
 }
 
+func CopyCoreFiles(pod, ns, source, dest string) {
+	// kubectl cp <some-namespace>/<some-pod>:/tmp/foo /tmp/bar
+	cmd := fmt.Sprintf("kubectl cp %s/%s:%s %s",
```

Comment:
probably needs -c istio-proxy ?

Reply:
I think 'cp' doesn't need -c, there is a single filesystem namespace for all pods.

Reply:
I'm basing this on the help string:
```
expected 'cp <file-spec-src> <file-spec-dest> [-c container]'.
```